org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/scoping/batch/StaticExtensionFeatureDescriptionWithImplicitFirstArgument.java

```diff
@@ -18,7 +18,10 @@
  */
 public class StaticExtensionFeatureDescriptionWithImplicitFirstArgument extends StaticExtensionFeatureDescription {
 
-	protected StaticExtensionFeatureDescriptionWithImplicitFirstArgument(
+	/**
+	 * @since 2.17
+	 */
+	public StaticExtensionFeatureDescriptionWithImplicitFirstArgument(
```

Comment:
@szarnekow should we go this way or have a factory method?

Reply:
I think it's fine this way. We should probably do this for the other discriptions, too.